setup.py

```diff
@@ -60,4 +60,8 @@
         'scipy == 0.17.1',
     ],
 
+    # pytest integration
+    setup_requires=['pytest-runner'],
```

Comment:
I added an explicit pytest version to the install_requires list. The pytest-runner now does not need to have an explicit version.